api/openapi-spec/v3/.well-known__openid-configuration_openapi.json

```diff
@@ -15,7 +15,7 @@
   },
   "openapi": "3.0.0",
   "paths": {
-    "/.well-known/openid-configuration/": {
+    "/.well-known/openid-configuration": {
```

Comment:
this change is questionable, is there a way to avoid a user-facing change in our served openapi here?